src/benchmark/summarizer/second-phase/graph.ts

```diff
@@ -56,10 +56,10 @@ export function writeGraphOutput(ultimate: UltimateSlicerStats, outputGraphPath:
 	})
 	data.push({
 		name:  'memory (df-graph)',
-		unit:  'Bytes',
-		value: Number(ultimate.dataflow.sizeOfObject.mean),
-		range: Number(ultimate.dataflow.sizeOfObject.std),
-		extra: `median: ${(ultimate.dataflow.sizeOfObject.median).toFixed(2)}`
+		unit:  'KiB',
```

Comment:
can you please retroactively update all existing data elements in the json so that we get a uniform representation?

Reply:
do u want me to suffer